### PR TITLE
Adding serverSecurityAlertPolicies.json from pr

### PR DESCRIPTION
**Pull request by yaakoviyun**
Mobing the API swagger as it is GA.

See PR for the private repo:
https://github.com/Azure/azure-rest-api-specs-pr/pull/412

### CODE REVIEW

**Comment by yaakoviyun** [comment]
@ayeletshpigelman - please review too

**Comment by sarangan12** [comment]
Assigning it to @marstr as he reviewed the original private pr

**Review by jaredmoo — APPROVED** [review]

**Comment by yaakoviyun** [comment]
@marstr , can you please merge?

**Comment by yaakoviyun** [comment]
@marstr , can you please merge?

**Comment by marstr** [comment]
Howdy @yaakoviyun,

I'm restarting the required Python job, because the actual failure looked like it was due to a timeout.

If that passes, I'll merge this afternoon.

**Review by marstr — CHANGES_REQUESTED** [review]
Looks like the python job is now catching a duplicated parameter: https://travis-ci.org/Azure/azure-rest-api-specs/jobs/387625721#L674

**Comment by yaakoviyun** [comment]
@jaredmoo - can you help. It seems like this error is for every swagger file

**Comment by jaredmoo** [comment]
Compare the definition of "SubscriptionIdParameter" in your file with one of the others (e.g. `preview/2017-03-01-preview/serverDnsAliases.json`) and you will see there is a difference. Try fixing that.

**Comment by jaredmoo** [comment]
BTW you can see the repro command line here:  https://travis-ci.org/Azure/azure-rest-api-specs/jobs/387625721#L665

`autorest /git-restapi/specification/sql/resource-manager/readme.md --multiapi --python --python-mode=update --python-sdks-folder=/tmp/tmpo3t0d1r9/sdk --use=@microsoft.azure/autorest.python@~3.0 --version=preview`

**Comment by yaakoviyun** [comment]
Thanks.

I sent a fix.

Regards,
Yaakov

**Comment by jaredmoo** [comment]
Still failing: https://travis-ci.org/Azure/azure-rest-api-specs/jobs/388841512#L672

**Comment by yaakoviyun** [comment]
@marstr @jaredmoo - I've fixed all the issue.

File: specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01/databaseSecurityAlertPolicies.json [hunk 1]
```diff
@@ -181,7 +181,6 @@
           "description": "Specifies the state of the policy. If state is Enabled, storageEndpoint and storageAccountAccessKey are required.",
           "type": "string",
           "enum": [
-            "New",
```
Comment (marstr) [inline specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01/databaseSecurityAlertPolicies.json]: I hadn't noticed before, but this is a pretty real breaking change. Is it just unused, or is there something nefarious about including this? If it's simply not useful, I'd recommend continuing to include it, and just have the server respond with an appropriate error message and documenting somewhere that this is deprecated.

**Review by marstr — CHANGES_REQUESTED** [review]
One more breaking change that I suspect is avoidable, but other than that this LGTM.

**Comment by jaredmoo** [comment]
Even if it is deprecated, removing the value from Swagger will remove it from generated clients which can cause compile errors in customers apps. This is a breaking change and we must avoid it. Breaking changes must only be done when there is a strong practical reason.

**Comment by yaakoviyun** [comment]
@marstr  - fixed all issues.
can you merge?

**Review by marstr — APPROVED** [review]

**Comment by marstr** [comment]
These changes have already been approved by the ARM team in the private repository PR.

**Pull request merged**